src/ui/Button/Button.module.scss

```
@@ -20,11 +20,11 @@
 // icon
 .suffix-icon{
```

```suggestion
.suffix-icon {
```

После селектора всегда отступ перед `{`

---

done